src/main/java/com/faforever/client/main/event/LocalReplaysChangedEvent.java

```diff
@@ -2,12 +2,27 @@
 
 import com.faforever.client.replay.Replay;
 import lombok.Value;
+import org.springframework.context.ApplicationEvent;
 
 import java.util.Collection;
 
-@Value
-public class LocalReplaysChangedEvent {
+public class LocalReplaysChangedEvent extends ApplicationEvent {
```

Comment:
Extending Application Event is not needed

Reply:
I mean u can for sure it won't hurt, we did not do this in any other cases :thinking: not sure whats best here